Tests/FluentAssertions.Specs/Formatting/FormatterSpecs.cs

```diff
@@ -13,8 +11,8 @@
 
 namespace FluentAssertions.Specs.Formatting;
 
-[Collection("FormatterSpecs")]
-public class FormatterSpecs
+[Collection("FormattingSpecs")]
```

Comment:
To match the name of the `[CollectionDefinition]` on `FormatterSpecsDefinition`
```suggestion
[Collection("FormatterSpecs")]
```